### PR TITLE
Make examples more consistent

### PR DESCRIPTION
**Pull request by nickmccurdy**
## Changes

- [x] [Don't ignore .output](https://github.com/withastro/astro/commit/eef404df14b6a967a405e5e0ad92cebe849f514f)
- [x] [Make example readmes more consistent](https://github.com/withastro/astro/commit/cca2f846d6dc9c11dae82d0267e69a9138f87be9)
- [ ] I've noticed some examples are missing `check` scripts. Is that something we'd want to add?

## Testing

Diffing directories in examples to look for outdated files

## Docs

This doesn't change functionality, only fixing configs and docs.

### CODE REVIEW

**Review by sarah11918 — COMMENTED** [review]
All the consistency changes in the READMEs look good to me! But, as you've poined out, you've added `astro check` as an example command when isn't always defined in the project. (Doesn't look to be in the "basics" template, for example.) We do have a CLI command to automatically launch the docs; not sure whether that might be an alternative to consider? https://docs.astro.build/en/reference/cli-reference/#astro-docs

I'll leave the question as to whether to add the script to the other examples to someone else, but only request that you double check to make sure you do *not* use `astro check` as a README example if it isn't in the project itself.

File: examples/deno/README.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
-# Welcome to [Astro](https://astro.build)
+# Astro Starter Kit: Deno
 
-[![Open in StackBlitz](https://developer.stackblitz.com/img/open_in_stackblitz.svg)](https://stackblitz.com/github/withastro/astro/tree/latest/examples/basics)
+```
+npm create astro@latest -- --template deno
+```
+
+ [![Open in StackBlitz](https://developer.stackblitz.com/img/open_in_stackblitz.svg)](https://stackblitz.com/github/withastro/astro/tree/latest/examples/basics)
```
Comment (bluwy) [inline examples/deno/README.md:7]:
```suggestion
 [![Open in StackBlitz](https://developer.stackblitz.com/img/open_in_stackblitz.svg)](https://stackblitz.com/github/withastro/astro/tree/latest/examples/deno)
```

I believe the link here should be for deno.

**Comment by bluwy** [comment]
> But, as you've poined out, you've added `astro check` as an example command when isn't always defined in the project. (Doesn't look to be in the "basics" template, for example.) We do have a CLI command to automatically launch the docs; not sure whether that might be an alternative to consider? https://docs.astro.build/en/reference/cli-reference/#astro-docs

Looks like the docs template is the only one that has the `npm run check` command. Maybe we should remove that since it's invoking `tsc` but typescript isn't directly installed (which feels a bit wrong). I feel like recommending `astro check` sounds nice though, I actually didn't know typechecking is builtin by default.

**Comment by matthewp** [comment]
@nickmccurdy are you planning on updating with the requested changes?

**Comment by natemoo-re** [comment]
Cleaned up these changes, this should be good to merge now!

**Review by sarah11918 — COMMENTED** [review]
Still the issue that `check:` is not defined in the Deno starter's `package.json`, but we're suggesting adding it to the README as an example of a script you can run? 

Otherwise, no concerns!

**Comment by natemoo-re** [comment]
Ah! I just understood that whole convo.

We have an `"astro"` command defined in the README so that you can run `npm run astro add` and `npm run astro check` without needing to add an associated `package.json` script for every subcommand of Astro.

This is already the copy in all the other READMEs, so maybe a follow-up task to update?

**Review by sarah11918 — APPROVED** [review]

**Pull request merged**